src/HeuristicCompletion-Model/CoResultSet.class.st

```diff
@@ -117,7 +118,8 @@ CoResultSet >> initialize [
 
 	super initialize.
 	results := OrderedCollection new.
-	filter := CoFilter empty
+	filter := CoFilter empty.
+	sorter := CompletionContext sorterClass new
```

Comment:
Here this introduces a circular dependency... Maybe the SorterClass et al should be moved to this package?